### PR TITLE
Dms

### PR DESCRIPTION
**Pull request by clovellbsc**
## Before you start

1. Have you run `git pull origin main` on your branch to bring in the latest changes?
2. Is there an issue on the repository for this request? If not, please create one.
3. Have you followed the correct workflow in the [team documentation](https://docs.google.com/document/d/13yi709ax4nmfTG9J0F57yEFpYRH7fdtczwvVJCrNIsM/edit#)?
4. Conflict resolution to be dealt with **on GitHub**.

## What do these code changes do?
Added direct messaging functionality between friends

## How do these code changes affect the application?
Implemented new css styling in the style sheet.
Updated the nav-bar
added message controller
added message route
added message index hbs file
added message show hbs file
added cypress testing for messages
added jest testing for messages

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [x] I have added thorough tests.
- [x] I have created a PR ticket on the [Trello board](https://trello.com/b/P8lEsvZe/acebook).
- [x] I will link the PR to the Trello card once created.
- [x] I will send the PR in the Slack channel and ping @here.

### CODE REVIEW

**Comment by clovellbsc** [comment]
[Direct messaging implementation](https://trello.com/c/Wvk032AU/101-direct-messaging-implementation)

**Review by r94o — APPROVED** [review]
Excellent work - smashed it!

File: cypress/integration/messages_spec.js [hunk 1]
```diff
@@ -0,0 +1,168 @@
+describe("Messages", () => {
+  it("loads a blank message section with choose a conversation on the index", () => {
+    cy.signUp();
+    cy.login();
+
+    cy.get("#dropdownUser1").click();
```
Comment (PKilgarriff) [inline cypress/integration/messages_spec.js:6]: This has now changed to dropdownSessionUser FYI

**Review by PKilgarriff — APPROVED** [review]
Excellent work Chris, this was a huge feature - I hope you've bumped the effort class all the way up to max to reflect that! Just a watch-out on those tests that use dropdownUser1 as an id, we'll catch it on the next loop.

**Pull request merged**